### PR TITLE
[Doc] Update with Python 3.10 support

### PR DESCRIPTION
**Pull request by qiao-bo**
Related issue = #3985

v0.9.2 has been released with Python 3.10 support. In this PR we update the docs and finalize the steps for Taichi Python 3.10 support.

### CODE REVIEW

File: docs/lang/articles/misc/install.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ sidebar_position: 0
   ERROR: No matching distribution found for taichi
   ```
 
-  - Make sure you're using Python version 3.6/3.7/3.8/3.9:
+  - Make sure you're using Python version 3.6/3.7/3.8/3.9/3.10:
 
     ```bash
     python3 -c "print(__import__('sys').version[:3])"
-    # 3.6, 3.7, 3.8 or 3.9
+    # 3.6, 3.7, 3.8, 3.9, or 3.10
```
Comment (rexwangcc) [inline docs/lang/articles/misc/install.md:42]:
```suggestion
    # 3.6, 3.7, 3.8, 3.9 or 3.10
```

**Review by rexwangcc — APPROVED** [review]
LGTM, thanks!

File: docs/lang/articles/misc/install.md (hunk 1 above)
Comment (qiao-bo) [reply docs/lang/articles/misc/install.md:42]: @rexwangcc Thanks, I actually vote for keeping the [Oxford Comma ](https://prowritingaid.com/Oxford-Comma). ;/

Comment (rexwangcc) [reply docs/lang/articles/misc/install.md:42]: Cool, I cannot argue with that :P

**Pull request merged**